server/src/builder.rs

```diff
@@ -258,15 +259,18 @@ async fn build_edge(args: &EdgeArgs, app_name: &str) -> EdgeResult<EdgeInfo> {
         unleash_client: unleash_client.clone(),
         persistence: persistence.clone(),
     });
-
+    let feature_config = FeatureRefreshConfig::new(
+        Duration::seconds(args.features_refresh_interval_seconds as i64),
+        args.streaming,
+        args.strict,
```

Comment:
I think this is wrong. The argument order is strict first, then streaming.

So yeah, can we model this differently? E.g. an enum that's `dynamic  | strict | streaming`? (although they're not all edge modes, they cover all the cases here. It also prevents the "strict = false, streaming = true" issue, which should be invalid, right?